drivers/ethernet/phy/phy_microchip_ksz8081.c

```diff
@@ -289,20 +293,27 @@ static int phy_mc_ksz8081_reset(const struct device *dev)
 		goto done;
 	}
 
-	/* Wait for 500 ms as specified by datasheet */
-	k_busy_wait(USEC_PER_MSEC * 500);
+	/* Wait for at least 500 us as specified by datasheet */
+	k_busy_wait(1000);
 
 	/* Reset over */
 	ret = gpio_pin_set_dt(&config->reset_gpio, 1);
+
+	/* After deasserting reset, must wait at least 100 us to use programming interface */
+	k_busy_wait(200);
```

Comment:
wrong comment or wrong delay in code :)

Reply:
yeah i just doubled the minimum time arbitrarily but on purpose